.github/workflows/ansible-release.yml

```diff
@@ -0,0 +1,84 @@
+name: ansible-release
```

Comment:
No need to repeat the default value with is the filename. Normally, this field is for human-readable title that shows up in the GH UI.